Add iserv.host for IServ GmbH

KimBrodowski opened the pull request:
# Public Suffix List (PSL) Submission

<!--
Each PSL PR needs to have a description, rationale, indication of DNS validation and syntax checking, as well as a number of acknowledgements from the submitter. This template must be included with each PR, and the submitting party MUST provide responses to all of the elements in order to be considered.
-->

<!-- #### READ THIS FIRST ####

If you haven't yet, please read our guidelines:
https://github.com/publicsuffix/list/wiki/Guidelines#submit-the-change

Also, read them again, as many skip that part and 
get confused about why their PR is delayed or does
not get accepted when their submission didn't follow them.

A recent PR using the current template is 
https://github.com/publicsuffix/list/pull/1591, although 
the organization and description were not as substantial 
as desired, which required maintainers time to visit the 
requestor's website to further research.
Having more robust org/desc improves the PR processing 
pace due to the extra cycles not being lost to research.
For an example of what an excellent description in a PR looks like
see https://github.com/publicsuffix/list/pull/615, 
although that example uses an earlier template.
-->

### Checklist of required steps

* [x] Description of Organization
* [x] Robust Reason for PSL Inclusion
* [x] DNS verification via dig

* [x] Each domain listed in the PRIVATE section has and shall maintain at least two years remaining on registration, and we shall keep the `_psl` TXT record in place in the respective zone(s).

Note: See "Reason for PSL Inclusion" below! Impossible at this time for existing entries.

__Submitter affirms the following:__ 
<!--
Third-party Limits are used elsewhere, such as at Cloudflare, Let's 
Encrypt, Apple, GitLab or others, and having an entry in the PSL alters 
the manner in which those third-party systems or products treat 
a given domain name or sub-domains within it.

To be clear, it is appropriate to address how those limits impact 
your domain(s) directly with that third-party, and it is inappropriate 
to submit entries to the PSL as a means to work around those limits or 
restrictions.
-->

 * [x] We are listing *any* third-party limits that we seek to work around in our rationale such as those between IOS 14.5+ and Facebook (see [Issue #1245](https://github.com/publicsuffix/list/issues/1245) as a well-documented example)

<!--
The purpose of the question above is to expose limit workarounds.
If there are third party limits that the PR seeks to overcome, those
must be listed within the rationale section of this request, and 
provide a good level of detail the effort that was made to work directly 
with the third part(y|ies) in attempting to address this within their 
rationale response below.
In all cases, software and services should be discouraged from use of
the PSL as a rate-limiting tool, and provide clear instructions to their
own clients, partners and users on the manner in which they can directly
request rate limit increases.
We treat the following as an attestation in the public record of the 
requesting party that they are not attempting to bypass rate limits through
the PR.
-->

 * [x] This request was _not_ submitted with the objective of working around other third-party limits.

<!--
Submitter will maintain domains in good standing or may lose section.

The ongoing trust of the PSL requires it to be free of outdated or problematic entries. In making this pull request, there is a commitment by the submitter that they are going to review and maintain their relevant section. By submitting an entry, the requestor acknowledges that their entry and section may be removed if the domain does not maintain the respective _psl entries in DNS, any domain(s) within their section fail to resolve in DNS, the domain does not get renewed, expires or is otherwise unreachable. The submitter further identifies that it is their responsibility to review their submitted section within the PSL, submitting updates or removals as their domain(s) may change over time. It is also the responsibility of the submitter to provide (and keep up to date) a reachable email address within the section, and to maintain that address as it may change over time, so that they receive notices.
-->

 * [x] The submitter acknowledges that it is their responsibility to maintain the domains within their section. This includes removing names which are no longer used, retaining the _psl DNS entry, and responding to e-mails to the supplied address. Failure to maintain entries may result in removal of individual entries or the entire section.

<!--
The guidelines describe which section to place the entry, what the 
order of commented org placement, order of sorting of entries. 
(hint: TLD then SLD, ascending sorting) Although it seems pedantic, 
the sorting and formatting rules help ensure all of the automation 
that uses the PSL operates correctly. Typically both are solved or
neither.
-->

 * [x] The [Guidelines](https://github.com/publicsuffix/list/wiki/Guidelines) were carefully _read_ and _understood_, and this request conforms to them.
 * [x] The submission follows the [guidelines](https://github.com/publicsuffix/list/wiki/Format) on formatting and sorting.

<!-- 
Sorting and formatting of the entries is outlined in the guidelines 
and non-conforming requests are one of the largest sources of delay,
so getting this right initially will aid successfully having it 
proceed. Mislocated entries and trailing spaces should be avoided.
-->

<!--
In your submission, please include a role-based email address (e.g. security@example.com) rather than a personal email address (e.g. jane.doe@example.com) under your organization name, so that we can maintain contact with your organization, independent of personnel changes.
This email address will be used for any required verification or inquiries regarding your PSL listing. This inbox must be actively maintained and monitored for future communications from the PSL project for as long as the domain remains in the PSL. Any PSL inquiries sent to this address must receive a response within 30 days, as maintaining timely communication is required for continued inclusion in the PSL.
-->

 * [x] A role-based email address has been used and this inbox is actively monitored with a response time of no more than 30 days.

**Abuse Contact:** info@iserv.de

<!--
Please confirm that you have accessible abuse contact information on your website. 

At a minimum, you must provide an abuse contact either in the form of an email address or a web form that can be used to report abuse. This contact should be easily accessible to allow concerned parties to notify the registry or subdomain operator directly when malicious activities such as phishing, malware, or abuse are detected. For example, if you provide subdomains at example.com, where users may register subdomains such as clientname.example.com, then in case of abuse, reporters should be able to visit example.com and easily find the relevant abuse contact information.
-->

* [x] Abuse contact information (email or web form) is available and easily accessible.

  URL where abuse contact or abuse reporting form can be found:  https://iserv.de/impressum
  <!-- Provide the URL where an Internet user can access the abuse contact information -->

---

For PRIVATE section requests that are submitting entries for domains that match their organization website's primary domain, please understand that this can have impacts that may not match the desired outcome and take a long time to rollback, if at all.

To ensure that requested changes are entirely intentional, make sure that you read the affectation and propagation expectations, that you understand them, and confirm this understanding. 

PR Rollbacks have lower priority, and the volunteers are unable to control when or if browsers or other parties using the PSL will refresh or update.

<!-- 
Seriously, carefully read the downline flow of the PSL and the 
guidelines. Your request could very likely alter the cookie and 
certificate (as well as other) behaviours on your core domain name in 
ways that could be problematic for your business.

Rollbacks are really not predictable, as those who use or incorporate 
the PSL do what they do, and when. It is not within the PSL volunteers' 
control to do anything about that. 

The volunteers are busy with new requests, and rollbacks are lowest 
priority, so if something gets broken by your PR, it will potentially 
stay that way for an indefinite period of time (typically long).
-->

(Link: [about propagation/expectations](https://github.com/publicsuffix/list/wiki/Guidelines#appropriate-expectations-on-derivative-propagation-use-or-inclusion))

 * [x] *Yes, I understand*. I could break my organization's website cookies and cause other issues, and the rollback timing is acceptable. *Proceed anyways*.
---

<!--
As you complete each item in the checklist please mark it with an X.

For example:
* [x] Description of Organization
-->

## Description of Organization

IServ GmbH is a German company providing IT solutions for the educational sector. Our main product is also called IServ. The IServ school platform is an all-in-one solution for the pedagogical side of a school's network. It provides basic functionality like e-mail, file sharing, instant messaging, calendars, a collaborative text editor etc., but also software deployment and an Active Directory domain controller as well as more education oriented features such as special examination modes for local computers and an exercise module in over 50 modules.

IServ is currently used by over 6,000 schools primarily in German-speaking regions and manages about 540,000 computers and 5,000,000 user accounts.

I'm the chief information security officer and an engineer at IServ GmbH. Previous requests for our organization have been submitted by me or my co-worker Mario Hoberg, a senior engineer at IServ GmbH.

**Organization Website:** https://iserv.de/

## Reason for PSL Inclusion

We deploy IServ on-premise and cloud hosted under unique customer domains. For technical reasons, we need a static hostname for installations that cannot change. Since customers can request their end-user visible domains to change, we registered iserv.host as a domain we can provide static aliases with.

Hostnames will have the format %UUID%.iserv.host and are used in the background to connect servers to each other.

Since customers have full control of their servers and DNS zones, we must isolate them from each other. This is why we would like the domain to be added to the PSL.

Some protocols, e.g. [Matrix](https://matrix.org/) will make use of this hostname in customer facing applications, so it's important to us that e.g. cookies remain separate.

Unfortunately, it is not possible for us at this time to maintain 1 year of validity and provide 2 years of renaming validity for our existing de domains on the list. Our registrar INWX does not allow us to prolong de domains beyond the point they are already prolonged as far as I can tell. The requirement introduces new monitoring challenges and hidden expiration dates while providing little benefit, since domains can be deleted at any point. Our domains all automatically renew. We can affirm that we will keep this in place if that helps. Of course, we will keep `_psl`   RRs in place, as we have with all previous submissions.

<!--
Please tell us why your domain(s) should be listed in the PSL
(i.e. Cookie Security, Let's Encrypt issuance, IOS/Facebook, 
Cloudflare, etc.) and clearly confirm that any private section 
names hold registration term longer than 2 years and shall 
maintain more than 1 year term in order to remain listed.

If you are attempting to work around third party limits, use 
this area to describe how and detail the manner in which you 
have first attempted to engage those third parties on the 
matter.

Please also reference any past issues or PRs 
specifically related to this submission or section.

Provide three or more sentences here that describe the purpose 
for which your PR should be included in the PSL. There is no 
upper limit, but six paragraphs seems like a rational stop.
-->

**Number of users this request is being made to serve:** 5,000,000


Previous requests:

* [Add \`iservschule.de\`, \`schulplattform.de\`, update IServ GmbH contact information by MarioHoberg · Pull Request #1580 · publicsuffix/list](https://github.com/publicsuffix/list/pull/1580)
* [Add schulserver.de, update IServ GmbH contact information by KimBrodowski · Pull Request #996 · publicsuffix/list](https://github.com/publicsuffix/list/pull/996)
* [Add iserv.dev for IServ GmbH by KimBrodowski · Pull Request #826 · publicsuffix/list](https://github.com/publicsuffix/list/pull/826)
* [Adding mein-iserv.de, test-iserv.de for IServ GmbH by KimBrodowski · Pull Request #552 · publicsuffix/list](https://github.com/publicsuffix/list/pull/552)

## DNS Verification
~~~
kab@kim-nb:~$ dig _psl.iserv.host TXT @8.8.8.8 +short
"https://github.com/publicsuffix/list/pull/2461"
~~~
<!--
For each domain you'd like to add to the list please create
a DNS verification record pointing to your pull request.

For example, if you'd like to add example.com and example.net
you would need to provide the following verifications:

```
dig +short TXT _psl.example.com
"https://github.com/publicsuffix/list/pull/XXXX"
```

```
dig +short TXT _psl.example.net
"https://github.com/publicsuffix/list/pull/XXXX"
```

Note that XXXX is replaced with the number of your pull request.

We ask that you leave this record in place while you want 
your entry to remain in the PSL, so that future (TBD) 
automation can remove entries where the record is not present.
-->

groundcat commented:
- [x] Expiration (Note: Must STAY >2y at all times)
  - [x] iserv.host 2028-04-25
- [x] Reasoning/Organization description
  - Organization description is thorough and explains their role in educational IT
  - Use case makes sense for PSL inclusion - they need cookie isolation between different customer environments
  - Not attempting to work around third-party limits
  - The request is for a company serving millions of educational users, which demonstrates sufficient scale to merit inclusion.
- [x] Non-personal email address
  - `info@iserv.de` is used, which is a non-personal email address
- [ ] Abuse contact
  - Abuse contact info available at https://iserv.de/impressum
  - **The domain iserv.host doesn't appear to have a working website. It would be helpful to have this domain redirect to your main website (iserv.de) or set up a basic landing page with contact information. This would improve transparency and make it easier for people to reach you if needed.**

KimBrodowski commented:
> The domain iserv.host doesn't appear to have a working website. It would be helpful to have this domain redirect to your main website (iserv.de) or set up a basic landing page with contact information. This would improve transparency and make it easier for people to reach you if needed.

I've added a redirect. TLS certificate is invalid at the moment. I've fixed that internally, but it'll have to go through review before it can go live.
